mlflow/tracking/fluent.py

```diff
@@ -557,6 +557,32 @@ def log_artifacts(local_dir, artifact_path=None):
     MlflowClient().log_artifacts(run_id, local_dir, artifact_path)
 
 
+def log_text(text, artifact_file):
```

Comment:
Should we mark this API as experimental?

Reply:
I think it's fine for this API to be stable; I don't imagine we'll want to rename or remove the `text` or `artifact_file` parameters

Reply:
Makes sense.